django/thunderstore/repository/api/v1/urls.py

```diff
@@ -15,6 +16,11 @@
 urls = [
     path("current-user/info/", CurrentUserInfoView.as_view(), name="current-user.info"),
     path("bot/deprecate-mod/", DeprecateModApiView.as_view(), name="bot.deprecate-mod"),
+    path(
```

Comment:
Placing the URL definition here seems to create two endpoints, one with the community as part of the path and second where it's presumably part of the subdomain. Is this still something we need (other than for backwards compatibility) or can we do just the community-in-the-path for new URLs from now on?

Reply:
Ideally we'd not include this API view twice, here's a related PR I attempted solving this issue in already but ran into a problem with some existing tests that test API url parity between the two forms: https://github.com/thunderstore-io/Thunderstore/pull/951

So primarily this is blocked by adjusting the test case so that it doesn't expect parity between the two URL structures. That PR I linked includes some noise in the diffs which have already been merged separately w/o the URL structure changes, the actual changes are much smaller

Reply:
@MythicManiac created https://github.com/thunderstore-io/Thunderstore/pull/1042